documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -95,6 +95,18 @@ include::rest_examples/get_v2_configuration.adoc[]
 |Sets the required format to return content. Supported formats are `application/xml` and `application/json`. The default is `application/json`. See link:#rest_accept[Accept] for more information.
 |===
 
+[[rest_v2_cache_config_convert]]
+=== Converting Cache Configurations
+
+To convert a certain existing cache configuration that is in XML format to JSON, invoke:
+
+[source,options="nowrap",subs=attributes+]
+----
+POST /rest/v2/caches?action=toJSON
```

Comment:
I don't understand the need for a special action. Couldn't this behaviour be triggered by the Accept-Content  header ?

Reply:
This  method is not touching any cache or existing config, it is converting something POSTed as XML to JSON

Reply:
Sorry I completely misunderstood that :)